dot/parachain/collator-protocol/validator_side_test.go

```diff
@@ -78,18 +93,16 @@ func TestProcessOverseerMessage(t *testing.T) {
 			errString:   ErrPeerIDNotFoundForCollator.Error(),
 		},
 		{
-			description: "ReportCollator message succeeds and reports a bad collator",
-			msg:         collatorprotocolmessages.ReportCollator(testCollatorID),
-			net: func() Network {
-				ctrl := gomock.NewController(t)
-				net := NewMockNetwork(ctrl)
-				net.EXPECT().ReportPeer(peerset.ReputationChange{
+			description:           "ReportCollator message succeeds and reports a bad collator",
```

Comment:
Use underscores instead of space in description test cases.

Reply:
Doesn't really matter. 
Underscore helps if you want to run individual tests, so you can just copy paste things. But it sorta looks ugly. An changing spaces to underscore at one place will create a precedence to do it everywhere.